rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -316,6 +317,25 @@ void deserializeRskTxHash_withNullValue_throwsIllegalArgumentException() {
         assertThrows(IllegalArgumentException.class, () -> deserializeRskTxHash(null));
     }
 
+    @Test
+    void serializeAndDeserializeReleaseOutpointsValues_shouldReturnExpectedValues() {
+        // arrange
+        List<Coin> outpointsValues = Arrays.asList(Coin.valueOf(12345), Coin.SATOSHI, Coin.COIN);
+
+        // Act
+        byte[] serialized = BridgeSerializationUtils.serializeOutpointsValues(outpointsValues);
+        List<Coin> deserialized = deserializeOutpointsValues(serialized);
```

Comment:
```suggestion
        List<Coin> deserialized = BridgeSerializationUtils..deserializeOutpointsValues(serialized);
```

Just to be consistent since the above static method uses the class reference

Reply:
updated